contribs/gnokeykc/main.go

```diff
@@ -13,7 +14,11 @@ import (
 func main() {
 	stdio := commands.NewDefaultIO()
 	wrappedio := &wrappedIO{IO: stdio}
-	cmd := client.NewRootCmd(wrappedio)
+	baseCfg := client.BaseOptions{
+		Home:   gnoenv.HomeDir(),
+		Remote: "127.0.0.1:26657",
+	}
```

Comment:
maybe we can use [`DefaultBaseOptions`](https://github.com/gnolang/gno/blob/2b53933cc28e3c7d650a136dc1b503b2864c27fb/tm2/pkg/crypto/keys/client/common.go#L11-L17) here? 
So we don't need to worry about modifying `Remote` at two places (just-in-case)
```suggestion
	baseCfg := client.DefaultBaseOptions
	baseCfg.Home = gnoenv.HomeDir()
```